CMakeLists.txt

```diff
@@ -104,6 +104,7 @@ option(AVX2 "Enable use of the Advanced Vector Extensions 2 (AVX2) instruction s
 option(AVX512 "Enable use of the Advanced Vector Extensions 512 (AVX512) instruction set" OFF)
 option(AVX512_SKYLAKE "Enable use of the Advanced Vector Extensions 512 (AVX512) instruction set targeting Skylake" OFF)
 option(LTO "Enable link time optimisations" OFF)
+OPTION(PREFETCH "Enables the -fprefetch-loop-arrays compiler flag for gcc" OFF)
```

Comment:
A bit of a side comment: Regarding PREFETCH and LTO: Is there any reason not to switch these on by default? Does it go any faster, and is it still correct?

Reply:
In my (brief) testing it was a little slower, but this was a long time ago and things have since changed. It is best to leave it off now. It might only be good to use if the compiling computer is the same as the one you are going to run it on, so having it on by default is a bad idea at a total guess.